webapp-shadcn/src/app/(dashboard)/layout.tsx

```
@@ -0,0 +1,44 @@
+      {/* Sidebar - Desktop */}
+      <div className="hidden lg:block">
+        <Sidebar isCollapsed={isCollapsed} />
+      </div>
+
+      {/* Sidebar - Mobile */}
+      {isMobileOpen && (
+        <>
+          <div
+            className="fixed inset-0 z-40 bg-black/50 lg:hidden"
+            onClick={() => setIsMobileOpen(false)}
+          />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)